style.css

```diff
@@ -211,6 +234,115 @@ a {
 	display: inherit;
 }
 
+/*--------------------------------------------------------------
+# Content
+--------------------------------------------------------------*/
+
+.section-inner.thin,
+.post-meta-wrapper,
+.post-nav-links,
+.author-bio,
+.entry-content > *:not(.alignwide):not(.alignfull):not(.alignleft):not(.alignright):not(.is-style-wide),
+[class*="__inner-container"] > *:not(.alignwide):not(.alignfull):not(.alignleft):not(.alignright):not(.is-style-wide),
+[class*="__inner-container"] .wp-block-group:not(.alignfull):not(.alignwide):not(.alignleft):not(.alignright),
+[class*="__inner-container"] .wp-block-cover:not(.alignfull):not(.alignwide):not(.alignleft):not(.alignright),
+.alignfull > figcaption,
+.alignfull > .wp-caption-text,
+.comment-respond p:not(.comment-notes) {
+	max-width: var( --default-content-width );
+}
+
+@media ( min-width: 660px ) {
+	.entry-content > p .alignleft,
+	.entry-content > .wp-block-image .alignleft,
+	figure.wp-block-gallery.alignleft {
+		margin-left: calc(( 100vw - var( --default-content-width ) - 8rem ) / -2);
+	}
+
+	.entry-content > p .alignright,
+	.entry-content > .wp-block-image .alignright,
+	figure.wp-block-gallery.alignright {
+		margin-right: calc(( 100vw - var( --default-content-width ) - 8rem ) / -2);
+	}
+}
+
+@media ( min-width: 1000px ) {
```

Comment:
Is there content or a specific page that you'd recommend testing these alignment styles against?

Reply:
Images are the best way to test here.